concent_api/core/tests/test_unit_validation.py

```diff
@@ -268,3 +271,50 @@ def test_that_if_frames_are_not_one_after_the_other_method_should_pass(self):
     def test_that_if_frames_are_not_integers_method_should_raise_exception(self):
         with self.assertRaises(FrameNumberValidationError):
             validate_frames(['1', '2'])
+
+
+class TestValidateComputeTaskDef(object):
+    compute_task_def = None
+
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.compute_task_def = ComputeTaskDefFactory()
+        self.compute_task_def["extra_data"] = {
+            "output_format": "PNG",
+            "scene_file": "nice_photo.blend",
+            "frames": [1, 2, 3],
+        }
+
+    def test_that_mising_extra_data_causes_message_validation_error(self):
+        del self.compute_task_def['extra_data']
+        with pytest.raises(MessageValidationError) as e:
```

Comment:
As in another files we don't use shortcuts, please use `exception` :)

Reply:
done